azure-pipelines-integration.yml

```diff
@@ -25,15 +25,16 @@ jobs:
       debug_32:
         _configuration: Debug
         _oop64bit: false
-      debug_64:
-        _configuration: Debug
-        _oop64bit: true
+      # 64-bit disabled for https://github.com/dotnet/roslyn/issues/40476
+      # debug_64:
+      #   _configuration: Debug
+      #   _oop64bit: true
```

Comment:
😨 These are our _only_ tests for OOP.

Reply:
64-bit testing covers the 99.9% customer scenario.

Reply:
@tmat and @jasonmalinowski confirmed that we were not introducing any new product bugs here, only exposing existing issues. Given that this was also blocking other work, we opted to not hold this PR hostage to existing bugs that cannot be fixed at the same time.

Reply:
And in parallel with that analysis, an image upgrade to our machine pool also meant the builds were entirely broken for integration tests _unless_ we merged this PR to upgrade to Preview 8. So this quickly went from something we were doing with regret to something we had no alternative for.

Reply:
The integration tests were working in this pull request as of afbbec1 (10 September). I've requested this be treated as proof of a ship-blocking product regression until we can figure the reason why they stopped working again.